metaflow/plugins/kfp/kfp_split_contexts.py

```diff
@@ -0,0 +1,178 @@
+import json
+import os
+import pprint
+from typing import Callable, Dict
+
+from metaflow import S3, FlowSpec, current
+from metaflow.datastore import MetaflowDataStore
+from metaflow.graph import DAGNode, FlowGraph
+from metaflow.plugins.kfp.kfp_constants import (
+    KFP_METAFLOW_CONTEXT_DICT_PATH,
+    PASSED_IN_SPLIT_INDEXES_ENV_NAME,
+    SPLIT_INDEX_SEPARATOR,
+)
+
+
+def graph_to_task_ids(graph: FlowGraph) -> Dict[str, int]:
```

Comment:
Would be useful to add in a docstring for this function too please.

Reply:
done